tools.yml

```
@@ -911,18 +945,27 @@ tools:
       - singularity
   toolshed.g2.bx.psu.edu/repos/iuc/deseq2/deseq2/.*:
     mem: 8
+  toolshed.g2.bx.psu.edu/repos/iuc/dexseq/dexseq/.*:
+    cores: 8
   toolshed.g2.bx.psu.edu/repos/iuc/dexseq/dexseq_count/.*:
     mem: 25
+  toolshed.g2.bx.psu.edu/repos/iuc/disco/disco/.*:
+    cores: 8
+    mem: 24
+    env:
+      DISCO_MAX_MEMORY: {mem}
```

```suggestion
      DISCO_MAX_MEMORY: "{mem}"
```
Needs double quotes, or the yaml parser will think it's a dict